src/TouristInfoCards.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const TouristInfoCards = props => {
+  return (
+    <div className="tourist-card">
+      {props.citiesName.map(city => {
+        return (
+          <div className="card">
```

Comment:
We are returning jsx in a map here. We need to give the first element a key prop. Look at other comment for details, or in your browser console.

The correct code is:
```
{props.citiesName.map((city, index) => {
	return (
	    <div key={index} className="card">
```

Reply:
Thanks Done!